### PR TITLE
SPOI-4941 Deprecate topic and zookeeper property from KafkaOperator and move them to Consumer.

### PR DESCRIPTION
**Pull request by tushargosavi**

### CODE REVIEW

File: contrib/src/main/java/com/datatorrent/contrib/kafka/KafkaConsumer.java [hunk 1]
```diff
@@ -91,7 +91,7 @@ public KafkaConsumer(SetMultimap<String, String> zks, String topic)
    */
   @NotNull
   @Bind(JavaSerializer.class)
-  protected SetMultimap<String, String> zookeeper;
+  protected SetMultimap<String, String> zookeeperMap;
```
Comment (siyuanh) [inline contrib/src/main/java/com/datatorrent/contrib/kafka/KafkaConsumer.java]: keep it default, so we don't need setter/getter for that anymore

File: contrib/src/main/java/com/datatorrent/contrib/kafka/KafkaConsumer.java [hunk 2]
```diff
@@ -507,4 +526,27 @@ public synchronized void setupStats(KafkaMeterStats stat){
     }
   }
 
+  protected SetMultimap<String, String> parseZookeeperStr(String zookeeper)
```
Comment (siyuanh) [inline contrib/src/main/java/com/datatorrent/contrib/kafka/KafkaConsumer.java]: This can be private

Comment (tushargosavi) [inline contrib/src/main/java/com/datatorrent/contrib/kafka/KafkaConsumer.java]: made this private

File: contrib/src/main/java/com/datatorrent/contrib/kafka/KafkaConsumer.java (hunk 1 above)
Comment (tushargosavi) [inline contrib/src/main/java/com/datatorrent/contrib/kafka/KafkaConsumer.java]: made it package public. verified that the property is not shown up in ui.

**Pull request merged**